### ISSUE TITLE
Permission denied for 'jrubyfx_cache'

### ISSUE THREAD

**Issue by dangnlg2** [jruby/jrubyfx#121]
OS : OS X Sierra
java version "1.8.0_102"
Java(TM) SE Runtime Environment (build 1.8.0_102-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.102-b14, mixed mode)
Latest version of jrubyfx (1.2.0)

I set out to jarify my custom jrubyFx application and am getting the following issue with permission denied to the '.jrubyfx_cache' folder. I tried 'chmod 777' but to no avail.

```
Exception running Application:
#<Errno::EACCES: Permission denied - /.jrubyfx_cache>
org/jruby/RubyDir.java:458:in `mkdir'
/Applications/MagicDraw/plugins/com.nomagic.magicdraw.automaton/engines/jruby-complete-1.7.22.jar!/META-INF/jruby.home/lib/ruby/1.9/fileutils.rb:247:in `fu_mkdir'
/Applications/MagicDraw/plugins/com.nomagic.magicdraw.automaton/engines/jruby-complete-1.7.22.jar!/META-INF/jruby.home/lib/ruby/1.9/fileutils.rb:221:in `mkdir_p'
org/jruby/RubyArray.java:1693:in `reverse_each'
/Applications/MagicDraw/plugins/com.nomagic.magicdraw.automaton/engines/jruby-complete-1.7.22.jar!/META-INF/jruby.home/lib/ruby/1.9/fileutils.rb:219:in `mkdir_p'
org/jruby/RubyArray.java:1613:in `each'
/Applications/MagicDraw/plugins/com.nomagic.magicdraw.automaton/engines/jruby-complete-1.7.22.jar!/META-INF/jruby.home/lib/ruby/1.9/fileutils.rb:205:in `mkdir_p'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/jrubyfx-fxmlloader-0.4.1-java/lib/fxmlloader/fxml_jit_info.rb:64:in `initialize'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/jrubyfx-fxmlloader-0.4.1-java/lib/jrubyfx-fxmlloader.rb:294:in `load'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/jrubyfx-1.2.0-java/lib/jrubyfx/controller.rb:313:in `load_fxml_only'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/jrubyfx-1.2.0-java/lib/jrubyfx/core_ext/stage.rb:80:in `fxml'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/CommonPlugins-2018.7.1.1/lib/CommonPlugins/swapfx.rb:107:in `start'
org/jruby/RubyBasicObject.java:1594:in `instance_eval'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/jrubyfx-1.2.0-java/lib/jrubyfx/module.rb:49:in `with'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/CommonPlugins-2018.7.1.1/lib/CommonPlugins/swapfx.rb:106:in `start'
/Applications/MagicDraw/plugins/G2/CommonPlugins_gemjar.jar!/gems/jrubyfx-1.2.0-java/lib/jrubyfx/java_fx_impl.rb:119:in `launch_app_after_platform'
org/jruby/gen/InterfaceImpl726911857.gen:13:in `run

```

**Comment by byteit101** [comment]
Hmm... that is odd, though it looks like it could occur if all your paths are jar relative but without the required `jar:` prefix. If this is still an issue, could you give me more details on the setup, specifically how you are going about jarifying?

**Comment by byteit101** [comment]
JRubyFX 2.0 has been released. Your issue may have been fixed in 2.0. If your issue still applies to JRubyFX 2.0, please reopen this issue.
